ui/src/pages/namespace/Instances/Detail/Main/Logs/index.tsx

```diff
@@ -117,6 +118,11 @@ const LogsPanel = () => {
           </TooltipProvider>
         </ButtonBar>
       </div>
+      <div className="mb-3">
```

Comment:
This was tricky to fix due to the grid layout and the requirements of the virtual scrolling container, but as discussed in person, it could be [solved](a87b50e49) by moving the alert component into the header of the grid. Thanks for the support!

Screenshots:
![image](https://github.com/direktiv/direktiv/assets/129740294/3c6f990a-a48c-4597-832e-b17fb9213f7d)
![image](https://github.com/direktiv/direktiv/assets/129740294/029f7504-0464-457e-909f-699a748d62a7)
![image](https://github.com/direktiv/direktiv/assets/129740294/c6210715-95fc-40c6-8254-2de3873acdc2)

Reply:
As discussed in person, this could be solved (a87b50e49) by moving the ApiError component into the header of the grid layout.
![image](https://github.com/direktiv/direktiv/assets/129740294/ad4d3399-f541-47e1-a46e-68b334c7fa84)
![image](https://github.com/direktiv/direktiv/assets/129740294/547c4ba0-3858-458b-8710-671c82f00180)
![image](https://github.com/direktiv/direktiv/assets/129740294/c246cbec-f4fa-4c9c-b663-93034e7bb883)